src/Build/Instance/TaskFactories/TaskHostTask.cs

```diff
@@ -70,9 +70,9 @@ internal class TaskHostTask : IGeneratedTask, ICancelableTask, INodePacketFactor
         private ConcurrentQueue<INodePacket> _receivedPackets;
 
         /// <summary>
-        /// The set of parameters used to decide which host to launch.
+        /// The structured task host parameters for better performance.
```

Comment:
return the original comment "/// The set of parameters used to decide which host to launch."